src/components/TopBar.js

```diff
@@ -22,16 +23,20 @@ class TopBar extends Component {
                 <Text type="basic" color={ Colors.white } display={ 'inline' }>
                     Zaloguj się przez GitHub
                 </Text>
-                <Button marginLeft='10px' onClick={this.onClick}
-                        type="primary">
+                <Button marginLeft='10px' onClick={this.onClick} type="primary">
```

Comment:
here `primary` in double quotes?

Reply:
To be honest I haven't even noticed, as my changes only re-formatted this part. Will check this within next PR. Good catch!